spec/factories/target_factory.rb

```diff
@@ -6,5 +6,6 @@
     title     { Faker::Friends.character }
     latitude  { Faker::Address.latitude }
     longitude { Faker::Address.longitude }
+    user      { create :user }
```

Comment:
You don't need to add the `{ create :user }`, just `user` is enough

Reply:
just user didn't work

Reply:
I meant this:
```
FactoryBot.define do
   factory :target do
     topic     { Faker::Book.genre }
     length    { Faker::Number.between(100, 1000) }
     title     { Faker::Friends.character }
     latitude  { Faker::Address.latitude }
     longitude { Faker::Address.longitude }

     user
  end
end
```
Does it work? If not, perhaps your current version of factory bot gem doesn't have this so don't worry

Reply:
Yes it did, thanks.